packages/sdk/src/entities/InlineTool.ts

```diff
@@ -23,6 +24,32 @@ export interface FormattingActionWithRange {
   range: TextRange;
 }
 
+/**
+ * @todo support fakeSelectionRequired option
+ * Interface that represents options handled by toolbar element
+ */
+export interface ToolbarOptions {
+  fakeSelectionRequired: boolean
```

Comment:
description missed

Reply:
there is todo for supporting this option, for now it just does nothing